crates/miden-testing/tests/auth/rpo_falcon_procedure_acl.rs

```diff
@@ -165,3 +169,138 @@ fn test_rpo_falcon_procedure_acl() -> anyhow::Result<()> {
 
     Ok(())
 }
+
+#[test]
+fn test_rpo_falcon_procedure_acl_with_allow_unauthorized_output_notes() -> anyhow::Result<()> {
+    let (component, auth_trigger_procedures, note) = setup_rpo_falcon_procedure_acl_test()?;
```

Comment:
Nit: To deduplicate a bit more, we can also make `setup_rpo_falcon_procedure_acl_test` take `allow_unauthorized_output_notes: bool, allow_unauthorized_input_notes: bool` and then move the `AccountBuilder` and `MockChain`-building parts into it as well.